sky/shell/shell.cc

```diff
@@ -190,5 +287,46 @@ void Shell::GetRasterizers(
   *rasterizers = rasterizers_;
 }
 
+void Shell::AddPlatformView(const base::WeakPtr<PlatformView>& platform_view) {
+  DCHECK(ui_thread_checker_ && ui_thread_checker_->CalledOnValidThread());
```

Comment:
The platform view could have been collected by this point. Maybe a validity check.